modules/backend/widgets/search/partials/_search.htm

```diff
@@ -3,18 +3,17 @@
         placeholder="<?= $placeholder ?>"
         type="text"
         name="<?= $this->getName() ?>"
-        value="<?= $value ?>"
+        value="<?= $value ?>"		
         data-request="<?= $this->getEventHandler('onSubmit') ?>"
+		data-request-complete="if($(this).val().length) {$(this).next().show()} else {$(this).next().hide()};"
         <?= !$searchOnEnter ? 'data-track-input' : '' ?>
         data-load-indicator
         data-load-indicator-opaque
         class="form-control <?= $cssClasses ?>"
         autocomplete="off" />
     <button class="clear-input-text" 
             value=""
-			data-request="<?= $this->getEventHandler('onSubmit') ?>"
-			data-load-indicator
-			data-load-indicator-opaque
-			data-request-success="$(this).prev().val('');"
+			style="<?php if($value == '') echo 'display:none' ?>"
```

Comment:
change to `style="<?= empty($value) ? 'display: none;' : ''; ?>"`

Reply:
done